server/tests/backends/fixtures/convert/to_text_pypika.json

```diff
@@ -0,0 +1,171 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake",
+    "athena_pypika"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "convert",
+        "columns": [
+          "price_per_l"
+        ],
+        "data_type": "text"
+      },
+      {
+        "name": "sort",
+        "columns": [
+          {
+            "column": "name",
+            "order": "asc"
+          }
+        ]
+      }
+    ]
```

Comment:
Could you please `select` the relevant columns at the end of the pipeline ? :pray:

Reply:
yup :+1: